### PR TITLE
Fix missing indicator for successful install

### LINKED ISSUE #80: Missing indicator for successful install
One thing that could be improved overall is that we never tell the user they may need to reload their environment (by opening a new terminal or `exec $SHELL` or something). If you run it without `-v`, it just silently exits (which is fine, though I would like some indication that it completed successfully by default...), and if you try to `nix-shell` right after that, it won't work and the user is left guessing if it actually completed, or if something went wrong. I'll make a ticket about this.

_Originally posted by @cole-h in https://github.com/DeterminateSystems/harmonic/pull/34#pullrequestreview-1199655057_

---

Basically, I think this issue is twofold:

1. We never notify the user that the install finished successfully
2. We never notify the user that they may need to restart their shell (either by opening a new terminal, `exec $SHELL`, or anything else)

Example screenshot where I fumble through this exact thing:

image

### PR DESCRIPTION
**Pull request by Hoverbear**
Closes #80 

image

### CODE REVIEW

File: src/cli/subcommand/install.rs [hunk 1]
```diff
@@ -139,6 +140,20 @@ impl CommandExecute for Install {
             }
         }
 
+        println!(
+            "\
+            {success}\n\
+            To use Nix, open a new terminal or run `{shell_reminder}`\n\
```
Comment (cole-h) [inline src/cli/subcommand/install.rs:146]:
```suggestion
            To get started using Nix, you may need to start a new shell, open a new terminal, or run `{shell_reminder}`\n\
```

Maybe something like this?

Comment (Hoverbear) [reply src/cli/subcommand/install.rs:146]: Ya I was trying to keep it short so it didn't take multiple lines. I can do this instead though (though I will remove that extra comma you suggested.)

Comment (cole-h) [reply src/cli/subcommand/install.rs:146]: Not my Oxford comma 😭

Comment (Hoverbear) [reply src/cli/subcommand/install.rs:146]: That's not an oxford comma though! It's not a list.

Comment (Hoverbear) [reply src/cli/subcommand/install.rs:146]: "You may need to" kinda doesn't make sense here since they definitely need to...

Comment (cole-h) [reply src/cli/subcommand/install.rs]: I think we're talking about different commas lol -- I didn't think you were talking about the first comma (`Nix, [...]`), but the last comma (`, or [...]`)😆 

Yeah, seems fine :)

**Review by cole-h — APPROVED** [review]

**Pull request merged**